sympy/assumptions/handlers/sets.py

```diff
@@ -770,3 +770,28 @@ def _(expr, assumptions):
     x = expr.args[0]
     if ask(Q.algebraic(x), assumptions):
         return ask(~Q.nonzero(x - 1), assumptions)
+
+# TranscendentalPredicate
+
+@TranscendentalPredicate.register_many(Exp1, Pi, GoldenRatio)
+def _(expr, assumptions):
+    return True
+
+
```

Comment:
Try to format things the same as how other predicates are handled: with one space instead of two here